libs/pando/ui-sheet/src/components/DocumentDisplay.tsx

```diff
@@ -28,16 +48,17 @@ export function DocumentDisplay({
         <TextSectionDisplay textDocument={document} />
       )
     case 'conditional':
-      return null //TODO:
-    // return (
-    //   <ConditionalDisplay
-    //     conditional={document}
-    //     hideDesc={hideDesc}
-    //     hideHeader={hideHeader}
-    //     disabled={disabled}
-    //     bgt={bgt}
-    //   />
-    // )
+      return (
+        <SetConditionalContext.Provider value={setConditional}>
+          <ConditionalDisplay
+            conditional={document.conditional}
```

Comment:
can put this in the context. or move setconditional to be a prop

Reply:
Why use a context here actually? If its to avoid writing out param over and over, just make the type of setConditional into a Type for easier re-writing